be/src/exprs/vectorized/binary_function.h

```diff
@@ -31,12 +31,12 @@ class ResultNopCheck {
 template <typename OP>
 class BaseBinaryFunction {
 public:
-    template <PrimitiveType LType, PrimitiveType RType, PrimitiveType ResultType>
-    static ColumnPtr vector_vector(const ColumnPtr& v1, const ColumnPtr& v2) {
+    template <PrimitiveType LType, PrimitiveType RType, PrimitiveType ResultType, typename... Args>
+    static ColumnPtr vector_vector(const ColumnPtr& v1, const ColumnPtr& v2, Args&&... args) {
```

Comment:
Why changed?

Reply:
DecimalV3Column::create(...) need extra precision and scale